base-math/truncddsf.c

```diff
@@ -41,9 +41,9 @@ CONVERT_WRAPPER(
 	_Decimal64 a_norm;
 	long long mant;
 	int exp, sexp;
-	
+
 	a_norm = getmantandexpd64 (a, &exp, 16, 1e16DD);
-	
+
```

Comment:
Why this whitespace cleanup in this pull request?

Reply:
I know need to open another pull request to it. However, I think we can fix it right here. I am sure that I will forget about that later.